pkg/apis/provisioning/v1alpha5/kubelet_configuration.go

```diff
@@ -22,4 +27,57 @@ type KubeletConfiguration struct {
 	// Note that not all providers may use all addresses.
 	//+optional
 	ClusterDNS []string `json:"clusterDNS,omitempty"`
+	// eventRecordQPS is the maximum event creations per second. If 0,
+	// there is no limit enforced.
+	EventRecordQPS *int32 `json:"eventRecordQPS,omitempty"`
+	// eventBurst is the maximum size of a burst of event creations,
```

Comment:
These comments need to match the variable name (in casing as well) in order for Go doc to understand it and be happy.

s/eventBurst/EventBurst